dotcom-rendering/src/web/components/Article.tsx

```diff
@@ -31,6 +31,10 @@ type Props = {
  * @param {ArticleFormat} props.format - The format model for the article
  * */
 export const Article = ({ CAPIArticle, NAV, format }: Props) => {
+	const showKeyEventsCarousel =
+		CAPIArticle.config.abTests.keyEventsCarouselVariant == 'variant' &&
+		CAPIArticle.config.switches.keyEventsCarousel;
```

Comment:
Wondering if we need this - if we need to remove the feature in a hurry would line 35 alone have the same effect as having line 36 as well?